modules/nf-core/calder2/main.nf

```diff
@@ -0,0 +1,43 @@
+def VERSION = '0.3'
+
+
+process CALDER2 {
+    tag '$meta.id'
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::r-calder2=0.3" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/r-calder2:0.3--r41hdfd78af_0' :
+        'quay.io/biocontainers/r-calder2:0.3--r41hdfd78af_0' }"
+
+
+    input:
+    tuple val(meta), path(cool)
+    val resolution
+
+    output:
+    tuple val(meta), path("${meta.id}/")            , emit: folder
+    path "versions.yml"                             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def suffix = resolution ? "::$resolution" : ""
+    def cpus = task.cpus ?: 1
+    """
+    # getting binsize
+    binsize="\$(cooler info --field bin-size $cool$suffix)"
+
+    echo "Detected binsize: \${binsize}"
+
+    # calling CALDER
+    calder --input $cool$suffix --outpath ${meta.id} --nproc $cpus --type cool --bin_size "\${binsize}" $args
```

Comment:
- Please split each parameter across multiple lines (see other modules as example)
- Please use `prefix` not `meta.id` for outpath, you will see examples for this in other modules [^1]


[^1]: https://nf-co.re/docs/contributing/modules#naming-conventions point 6